### PR TITLE
WIP: upgrading /legal to v1

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Upgraded the template markup in the /legal section of the site to Vanilla v1.

## Note

This is a large changeset as it was completed while header/footer QA was ongoing. I'm opening this PR so I can build a demo. We should discuss how best it can be reviewed.

## Demo

http://www.ubuntu.com-v1-legal-setup.demo.haus/legal

### CODE REVIEW

File: templates/legal/terms-and-policies/privacy-policy/third-parties.html [hunk 1]
```diff
@@ -7,66 +7,66 @@
 {% endblock second_level_nav_items %}
 
 {% block content %}
-<div class="row row-hero">
+<div class="row">
```
Comment (anthonydillon) [inline templates/legal/terms-and-policies/privacy-policy/third-parties.html]: Breadcrumbs and missing strip

File: templates/legal/terms-and-policies/thank-you.html [hunk 2]
```diff
@@ -7,18 +7,18 @@
 {% endblock second_level_nav_items %}
 
 {% block content %}
-<div class="row row-hero no-border">
+<div class="row">
```
Comment (anthonydillon) [inline templates/legal/terms-and-policies/thank-you.html]: Breadcrumbs update and missing strip

File: templates/legal/ubuntu-advantage/index.html [hunk 3]
```diff
@@ -3,41 +3,46 @@
...
 {% block content %}
+<div class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h1>Ubuntu Advantage</h1>
+      <p><a href="/support">Ubuntu Advantage</a> is Canonical&rsquo;s service package for Ubuntu. It offers tiered levels of support for desktop, server and cloud deployments. In this section, you can see our service description, but don&rsquo;t forget that
+        there will still be some details in your customer agreement that are specific to your deployment.</p>
+      <p>If you&rsquo;re interested in support for an Ubuntu deployment or you&rsquo;re a reseller and you want to offer it to your customers, you can <a href="http://ubuntu.com/management/ubuntu-advantage">learn more about Ubuntu Advantage here</a>.</p>
+    </div>
+    <div class="col-4">
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage/index.html]: Align center please

File: templates/legal/ubuntu-advantage/ua-terms-ja.html [hunk 4]
```diff
@@ -3,125 +3,126 @@
 {% block title %}UA 略式サービス契約{% endblock %}
 
 {% block second_level_nav_items %}
-	{% include "templates/_nav_breadcrumb.html" with section_title="Legal" subsection_title="Ubuntu Advantage" page_title="Service description" %}
+  {% include "templates/_nav_breadcrumb-v1.html" with section_title="Legal" subsection_title="Ubuntu Advantage" page_title="Service description" %}
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage/ua-terms-ja.html]: This page doesnt exist. its redirected to /legal/short-terms/ja

**Comment by anthonydillon** [comment]
We should also remove the legal section scss file and check the sass file is namespaced properly.

**Comment by barrymcgee** [comment]
@anthonydillon We're keeping all old SCSS files in place until all pages have been converted to v1.

**Comment by barrymcgee** [comment]
@anthonydillon Ready for re-review again

**Comment by anthonydillon** [comment]
Thanks, code and QA :+1:

**Comment by barrymcgee** [comment]
Superseded by https://github.com/canonical-websites/www.ubuntu.com/pull/1799

**Pull request closed without merging**